packages/vscode-extension/src/debug/prerequisitesHandler.ts

```diff
@@ -356,7 +411,7 @@ export async function checkAndInstall(): Promise<Result<any, FxError>> {
   return ok(null);
 }
 
-async function checkM365Account(prefix: string): Promise<CheckResult> {
+async function checkM365Account(prefix: string, showLoginPage = true): Promise<CheckResult> {
```

Comment:
Prefer making `showLoginPage` required, to ensure it's not misused.

Reply:
Good idea, added.